src/widgets/Dialogs.js

```diff
@@ -180,7 +180,7 @@ define(function (require, exports, module) {
             }
         } else if (brackets.platform === "mac") {
             // CMD+D Don't Save
```

Comment:
Hello,

Apple keyboards sometimes have only one key for deletions (that is, one key that acts as a backspace and delete key). This key is located on the upper right corner of the keyboard. For instance on my MacBook Pro its labeled 'Delete' and on my wireless Apple keyboard it is just labeled with a left pointing arrow ('<----'). So hitting CMD+Backspace/Delete triggers the Don't Save action.

CMD+D is also bind on the "Duplicate" action in Brackets and doesn't seem to trigger Don't Save at all under the Mac. 

I have tested the Don't Save behaviour on Apple's standard apps (like TextEdit, Preview) and CMD+D doesn't seem to trigger Don't Save either.
Whereas CMD+Backspace does.



Best Regards,
Lucas

> On 01 Feb 2015, at 15:13, Marcel Gerber <notifications@github.com> wrote:
> 
> In src/widgets/Dialogs.js <https://github.com/adobe/brackets/pull/10459#discussion_r23897756>:
> 
> > @@ -180,7 +180,7 @@ define(function (require, exports, module) {
> >              }
> >          } else if (brackets.platform === "mac") {
> >              // CMD+D Don't Save
> From what I can tell (not that I have a Mac...), it's Cmd+Delete:
>  <https://cloud.githubusercontent.com/assets/2641501/5992320/ba5a6c92-aa24-11e4-8bd8-00da4d6168f9.png>
> —
> Reply to this email directly or view it on GitHub <https://github.com/adobe/brackets/pull/10459/files#r23897756>.
>